src/components/SignerPanel/SignerPanelContent.js

```diff
@@ -137,9 +146,18 @@ class ActionPathsContent extends React.Component {
             You can directly perform this action:
           </DirectActionHeader>
         )}
-        <Info.Action icon={null} title="Action to be triggered:">
+        <Info.Action icon={null} title="Action to be triggered">
           {this.renderDescription(showPaths, intent)}
         </Info.Action>
+        {pretransaction && (
+          <Info.Action
+            title="Two transactions required"
+            style={{ marginTop: '20px' }}
+          >
+            This action requires two transactions to be signed in your Ethereum
+            provider, please confirm them one after another.
+          </Info.Action>
```

Comment:
Let's think about how to make the messaging around pretransactions better afterwards 💪!